hub/api/dataset.py

```diff
@@ -443,4 +468,171 @@ def __iter__(self):
                         cur[split_key[i]] = {}
                         cur = cur[split_key[i]]
                 cur[split_key[-1]] = torch.tensor(self._tensors[key][index])
-            yield (d)
\ No newline at end of file
+            yield (d)
+
+
+def from_tensorflow(ds):
+    """Converts a tensorflow dataset into hub format
+    Parameters
+    ----------
+    dataset:
+        The tensorflow dataset object that needs to be converted into hub format
+    Examples
+    --------
+    ds = tf.data.Dataset.from_tensor_slices(tf.range(10))
+    out_ds = dataset.from_tensorflow(ds)
+    res_ds = out_ds.store("username/new_dataset") # res_ds is now a usable hub dataset
+
+    ds = tf.data.Dataset.from_tensor_slices({'a': [1, 2], 'b': [5, 6]})
+    out_ds = dataset.from_tensorflow(ds)
+    res_ds = out_ds.store("username/new_dataset") # res_ds is now a usable hub dataset
+
+    ds = dataset.Dataset(schema=my_schema, shape=(1000,), url="username/dataset_name", mode="w")
+    ds = ds.to_tensorflow()
+    out_ds = dataset.from_tensorflow(ds)
+    res_ds = out_ds.store("username/new_dataset") # res_ds is now a usable hub dataset
```

Comment:
as far as I know, ds.store no longer existss, you should use commit, did you try running the examples?

Reply:
Oh ok I saw that it is in pipelines.